PWGHF/D2H/Tasks/taskDplus.cxx

```diff
@@ -213,6 +216,12 @@ struct HfTaskDplus {
         axesGenPrompt.insert(axesGenPrompt.end(), {thnAxisOccupancy});
         axesGenFD.insert(axesGenFD.end(), {thnAxisOccupancy});
       }
+      if (storePvContributors) {
+        axes.insert(axes.end(), {thnAxisPvContributors});
+        axesFD.insert(axesFD.end(), {thnAxisPvContributors});
+        axesGenPrompt.insert(axesGenPrompt.end(), {thnAxisPvContributors});
+        axesGenFD.insert(axesGenFD.end(), {thnAxisPvContributors});
```

Comment:
Why don't you just use `push_back`?

Reply:
I replaced these occurrences and a few others in the code, except in cases where multiple axes were added at once.